Garage 2.0/Views/Members/MemberOverviewIndex.cshtml

```diff
@@ -89,18 +86,20 @@
 
 <a asp-action="MemberOverviewIndex"
    asp-route-sortOrder="@ViewData["CurrentSort"]"
+   asp-route-currentFilter="@ViewBag.CurrentFilte"
    asp-route-pageNumber="@(Model.PageIndex - 1)"
    class="btn btn-default text-primary @prevDisabled">
     Previous
 </a>
 <a asp-action="MemberOverviewIndex"
    asp-route-sortOrder="@ViewData["CurrentSort"]"
+   asp-route-currentFilter="@ViewBag.CurrentFilte"
    asp-route-pageNumber="@(Model.PageIndex + 1)"
    class="btn btn-default text-primary @nextDisabled">
     Next
 </a>
-@if (ViewBag.Button != null)
+@*@if (ViewBag.Button != null)
 {
     <a asp-action="Index" class="btn btn-primary">Go Back</a>
-}
```

Comment:
The asp-action should be memberOverviewIndex insted of Index. I forgot to change that yesterday